test/e2e-encryption/encryption_test.go

```diff
@@ -33,11 +37,19 @@ func TestEncryptionTypeUnset(t *testing.T) {
 		EncryptionConfigSecretNamespace: operatorclient.GlobalMachineSpecifiedConfigNamespace,
 		OperatorNamespace:               operatorclient.OperatorNamespace,
 		TargetGRs:                       operatorencryption.DefaultTargetGRs,
-		AssertFunc:                      operatorencryption.AssertRoutesAndTokens,
+		AssertFunc:                      operatorencryption.AssertRoutes,
 	})
 }
 
 func TestEncryptionTurnOnAndOff(t *testing.T) {
+	ctx := context.Background()
```

Comment:
what is best practice for context in tests? Should we defer Cancel() ?

Reply:
changed to `context.TODO`